### ISSUE TITLE
Customize the number of input_channels in MobileNetv3_Large

### ISSUE THREAD

**Issue by ananda1996ai** [pytorch/vision#4332]
I would like to know how to customize the MobileNetV3_Large torchvision model to accept single-channel inputs with number of classes = 2.

As mentioned in some of the PyTorch discussion forums, I have tried
`model_ft.conv1 = nn.Conv2d(1, 64, kernel_size=7, stride=2, padding=3, bias=False)`
which works for ResNet models but not MobileNetV3.

Also tried,
`model.input_channels=1`

I keep getting the error 
`Given groups=1, weight of size [16, 3, 3, 3], expected input[12, 1, 512, 512] to have 3 channels, but got 1 channels instead`

Please provide suggestions for customizing the MobileNet model to accept single channel input, as is possible in the case of ResNet. Repeating my input tensor to have 3 channels is not a solution I would be interested in.

**Comment by datumbox** [comment]
@ananda1996ai There is neither `conv1` nor `input_channels` property on mobilenet. Your change has no effect.

You can try something like:
```
model.features[0][0] = nn.Conv2d(1, model.features[0][0].out_channels, kernel_size=3, stride=2, padding=1, bias=False)
```
